package.json

```diff
@@ -15,23 +15,39 @@
     "eslint-config-airbnb-base": "^11.1.3",
     "eslint-config-angular": "^0.5.0",
     "eslint-config-apiconnect": "^2.0.1",
+    "eslint-config-dbk": "^2.0.0",
+    "eslint-config-drupal": "^3.0.0",
     "eslint-config-ember": "^0.3.0",
+    "eslint-config-es5": "^0.5.0",
+    "eslint-config-es6": "^1.10.3",
     "eslint-config-google": "^0.7.1",
     "eslint-config-hapi": "^10.0.0",
     "eslint-config-jquery": "^1.0.1",
     "eslint-config-loopback": "^8.0.0",
     "eslint-config-nightmare-mode": "^2.3.0",
+    "eslint-config-nodesecurity": "^1.3.1",
     "eslint-config-prettier": "^1.5.0",
     "eslint-config-react-app": "^0.6.2",
+    "eslint-config-secure": "^0.2.1",
     "eslint-config-semistandard": "^11.0.0",
+    "eslint-config-signavio": "^3.2.0",
+    "eslint-config-signavio-test": "^2.0.0",
     "eslint-config-simplifield": "^5.0.1",
     "eslint-config-standard": "^10.0.0",
     "eslint-config-standard-jsx": "^4.0.0",
     "eslint-config-standard-react": "^5.0.0",
     "eslint-config-strongloop": "^2.1.0",
+    "eslint-config-vue": "^2.0.2",
+    "eslint-config-xo": "^0.18.2",
+    "eslint-config-xo-react": "^0.13.0",
+    "eslint-config-xo-space": "^0.16.0",
+    "eslint-import-resolver-node": "^0.3.1",
+    "eslint-import-resolver-webpack": "^0.8.3",
```

Comment:
We've had trouble with these revolvers in the past. Do we need to include them if we're blocklisting some of the `import/` rules?

Reply:
I think we do because the blocklisting happens when we report the issues. If we don't have these here, the analysis will error working with the config, right?

Reply:
Ahh, probably... sounds good to me.